pandas/core/base.py

```diff
@@ -1350,8 +1350,14 @@ def factorize(self, sort=False, na_sentinel=-1):
 
         Returns
         -------
-        indices : array of ints
-            Array of insertion points with the same shape as `value`.
+        int or array of ints
+            A scalar or array of insertion points with the
+            same shape as `value`.
```

Comment:
@datapythonista is this the correct format?

Reply:
looks good, but can you use `int or array of int` (I'd like to parse at some point the types from these, so I prefer the type name `int` over `ints`)